### PR TITLE
Add President's Choice Financial

### LINKED ISSUE #4045: President's Choice Financial Has some options
They support SMS, Phone call and email.
They have "forwarded the suggestion" of supporting authenticator apps/software tokens "to the correct department"

### PR DESCRIPTION
**Pull request by bobmatyas**
closes #4045

### CODE REVIEW

File: _data/banking.yml [hunk 1]
```diff
@@ -841,6 +841,15 @@ websites:
...
       lang: it
 
+    - name: President's Choice Financial
+      url: https://www.pcfinancial.ca
+      img: pcfinancial.png
+      tfa: Yes
+      sms: Yes
+      email: Yes
+      phone: Yes
+      doc: https://www.pcfinancial.ca/en/security
```
Comment (Carlgo11) [inline _data/banking.yml]: The doc link doesn't describe how to use or setup 2fa on the site so it's better just to leave it out.
```suggestion
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — DISMISSED** [review]

**Review by kmpoppe — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Comment by LukeAbellSmith** [comment]
Except PC Financial was already on twofactorauth.org, now there are 2, and they don't do banking.
image

**Comment by Carlgo11** [comment]
Hmm that shouldn't have happened 😕 
I'll remove the old one and move this one to the Finance category.

**Comment by Carlgo11** [comment]
Fixed in 139407606287a337c57ec4a893dbaca981875e97

**Pull request merged**